packages/turf-nearest-neighbor-analysis/README.md

```diff
@@ -30,7 +30,7 @@ Type: [object][1]
 
 ## nearestNeighborAnalysis
 
-Nearest Neighbor Analysis calculates an index based the average distances
+Nearest Neighbor Analysis calculates an index based on the average distances
```

Comment:
Looks like this was already in the index.ts comment, just not populated into the readme. I thought we had pre-commit hooks to make sure that didn't happen. Oh well.